ios/Sources/SettingFeature/NavigationBarConfigurator.swift

```diff
@@ -0,0 +1,53 @@
+import Styleguide
+import SwiftUI
+import UIKit
+
+public extension View {
```

Comment:
Is this extension looks better with Component or Utility module?

Reply:
I see, now this file has moved on to `Component` module.